posthog/models/person/person.py

```diff
@@ -138,17 +138,15 @@ class PersonOverride(models.Model):
 
     class Meta:
         constraints = [
-            models.UniqueConstraint(fields=["team", "old_person_id"], name="unique override per old_person_id"),
```

Comment:
`old_person_id` is a FK to a table that already includes `team`. By making it a `OneToOneField` below, it's not possible to have a duplicate `old_person_id`, period.